### ISSUE TITLE
Socket Error - Connect Error Message

### ISSUE THREAD

**Issue by SageKrow** [fireice-uk/xmr-stak-nvidia#97]
The full message is this:
SOCKET ERROR - CONNECT error: An attempt was made to access a socket in a way forbidden by its access permissions.

I'm currently running the cpu miner with no issue and wanted to add the gpu miner to maybe get a little bit more power. The pool I'm using is www.usxmrpool.com. I thought maybe the pool didn't like two connections from the same location on the same port and tried the 5555 port. I'm new to this so if there is any additional information that is needed, let me know.

Attached is the config file i am using (minus my wallet address). The numbers for the gpu my look weird because I found a few posts to decrease some of the numbers in order to fix a separate problem.
[config.txt](https://github.com/fireice-uk/xmr-stak-nvidia/files/1237304/config.txt)

**Comment by psychocrypt** [comment]
You need to add a xmr adress to you config.

```
[2017-08-20 21:46:46] : SOCKET ERROR - missing login
[2017-08-20 21:46:46] : SOCKET ERROR - RECEIVE error: socket closed
```

**Comment by SageKrow** [comment]
Are you referring to the pool address? there is an xmr address in the config, it's the same one the stak-cpu I'm running is using.
If your referring to the wallet address I removed it before uploading it to the post.

**Comment by SageKrow** [comment]
Did some more searching on google and I eventually found something.  Some people have been having issues with antivirus. For the most part it seems like mainly it quarantines the executable. However, I decided to take a look at mine and sure enough, it was blocking the executable from running due to generating 'suspicious internet activity'. I have BitDefender and I've since added the exception and the program runs fine now.
